test/validation.js

```diff
@@ -985,6 +995,41 @@ describe('validation tests', () => {
         .parse()
     })
 
+    it('fails with error if multiple inner commands are unknown', (done) => {
```

Comment:
Also while at it. Does this behaviour actually meaningful?
Wouldn't it be enough to only report the first unknown command, even if there are multiple?